addons/hr_holidays/views/hr_leave_views.xml

```diff
@@ -495,9 +495,6 @@
         <field name="mode">primary</field>
         <field name="priority">32</field>
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='all_employee_ids']" position="attributes">
-                <attribute name="invisible">1</attribute>
```

Comment:
The original purpose of this was to simply not show the employees on this view, as it is only "My time off" so we know the employee is me. However with viewpocalypse, this now needs to be column_invisible, so
``<attribute name="column_invisible">1</attribute>``

Reply:
> The original purpose of this was to simply not show the employees on this view, as it is only "My time off" so we know the employee is me. However with viewpocalypse, this now needs to be column_invisible, so `<attribute name="column_invisible">1</attribute>`

done.thanks